footylib/footylib.py

```diff
@@ -20,17 +21,17 @@ class Footy(object):
     def __init__(self):
         self.logger = logging.getLogger('{base}.{suffix}'.format(
             base=LOGGER_BASENAME, suffix=self.__class__.__name__))
-        self.site = "http://www.footy.eu/"
-        self.headers = {'User-Agent': 'Mozilla/5.0'}
+        self._site = "http://www.footy.eu/"
+        self._headers = {'User-Agent': 'Mozilla/5.0'}
```

Comment:
I liked the idea of variable just to be tidy but I guess it can always be if headers change in a future. Will do in next PR.